src/app.js

```diff
@@ -0,0 +1,29 @@
+import './assets/scss/app.scss';
+import $ from 'cash-dom';
+
+
+export class App {
+  initializeApp() {
+    let self = this;
+
+    $('.load-username').on('click', function (e) {
+      let userName = $('.username.input').val();
+
+      fetch('https://api.github.com/users/' + userName)
+        .then((response)=> {response.json})
+        .then(function (body) {
+          self.profile = body;
+          self.update_profile();
```

Comment:
This callback should be replaced with arrow function.
Arrow functions have lexical scope, therefore `this` will point to the instance of `App` class - this allows us to use `this` inside of the callback, instead of using `self`.